spec/controllers/idv/in_person/address_controller_spec.rb

```diff
@@ -3,21 +3,28 @@
 RSpec.describe Idv::InPerson::AddressController do
   include InPersonHelper
   let(:in_person_residential_address_controller_enabled) { true }
+  let(:usps_ipp_transliteration_enabled) { true }
   let(:pii_from_user) { Idp::Constants::MOCK_IPP_APPLICANT_SAME_ADDRESS_AS_ID_FALSE.dup }
   let(:user) { build(:user) }
   let(:flow_session) do
     { pii_from_user: pii_from_user }
   end
   let(:flow_path) { 'standard' }
+  let(:user_session) do
+    { idv: {} }
+  end
 
   before(:each) do
     allow(IdentityConfig.store).to receive(:in_person_residential_address_controller_enabled).
       and_return(true)
+    allow(IdentityConfig.store).to receive(:usps_ipp_transliteration_enabled).
+      and_return(true)
```

Comment:
You want to return `usps_ipp_transliteration_enabled` here, or delete the let statement for that if it's not needed.